tsdb/db_test.go

```diff
@@ -2668,3 +2668,27 @@ func TestChunkWriter(t *testing.T) {
 		})
 	}
 }
+
+func TestRangeForTimestamp(t *testing.T) {
+	type args struct {
+		t     int64
+		width int64
+	}
+	tests := []struct {
+		args     args
+		expected int64
+	}{
+		{args{1, 5}, 5},
+		{args{5, 5}, 10},
+		{args{6, 5}, 10},
+		{args{13, 5}, 15},
+		{args{95, 5}, 100},
+	}
+	for i, tt := range tests {
+		t.Run(strconv.Itoa(i), func(t *testing.T) {
```

Comment:
Usually we wouldn't use subtests here, these are all very fast tests. It usually only comes up with benchmarks.

Reply:
fixed.